installation/ilmt-track-bamlcns-metrictool.adoc

```diff
@@ -0,0 +1,42 @@
+= Tracking BAMOE License use in IBM License Metric Tool
+include::../partials/attributes.adoc[]
+//tag::ilmt-overview-description[]
+You can maintain an up-to-date inventory of software assets that are installed in your infrastructure, gather information about your hardware, and ensure license compliance of your enterprise by using ILMT. 
+You get to know the softwares that are existing in your system, and where or how it is used. 
+By monitoring consumption of PVU, VPC, and RVU MAPC metrics by IBM® products under full and subcapacity licensing terms, you can understand the current license consumption. 
+You can also predict the future license consumption, thus providing the means to optimize your license costs.
+//end::vscode-extensions-description[]
+
+== Configuring IBM License Metric Tool
+
+IBM® License Metric Tool generates reports to help you maintain compliance with your Virtual Processor Core (VPC) subcapacity license terms. 
+License Metric Tool can calculate VPC consumption only when software identification tags exist and are activated by the customer. 
+The optman global option licenseType must be set to perServer (default value) to activate Virtual Processor Core consumption tracking. 
```

Comment:
Typically we make the codes and source options stand out a bit more by using backticks:
The `optman` global option `licenseType` must be set to `perServer`...